[2025-04-05T15:55:35.091+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T15:55:35.128+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T15:55:32.315838+00:00 [queued]>
[2025-04-05T15:55:35.135+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T15:55:32.315838+00:00 [queued]>
[2025-04-05T15:55:35.136+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T15:55:35.147+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-05 15:55:32.315838+00:00
[2025-04-05T15:55:35.154+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=536) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T15:55:35.154+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'manual__2025-04-05T15:55:32.315838+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpbga2e7xe']
[2025-04-05T15:55:35.155+0000] {standard_task_runner.py:63} INFO - Started process 561 to run task
[2025-04-05T15:55:35.156+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask upload_to_minio
[2025-04-05T15:55:35.174+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T15:55:35.218+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T15:55:32.315838+00:00 [running]> on host a4017603cc0b
[2025-04-05T15:55:35.310+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T15:55:32.315838+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T15:55:32.315838+00:00'
[2025-04-05T15:55:35.311+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T15:55:35.325+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T15:55:35.326+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T15:55:35.327+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T15:55:35.341+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=manual__2025-04-05T15:55:32.315838+00:00, execution_date=20250405T155532, start_date=20250405T155535, end_date=20250405T155535
[2025-04-05T15:55:35.371+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T15:55:35.401+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T15:55:35.404+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
